client/src/components/Profile.jsx

```diff
@@ -130,6 +113,7 @@ function Profile({
       <p>Email: {userData.email}</p>
       <p>City: {userData.city}</p>
       <Button
+        data-testid="edit"
```

Comment:
Will you be adding a script to remove the data-test attributes and removing console.logs for production env?